src/view/app/FlameGraph/profilerFlameGraph.tsx

```diff
@@ -127,6 +167,34 @@ function ProfilerFlameGraph({
     });
   };
 
+  const openFileForFlameGraph = (node: FlameGraphNode): void => {
+    const foundModule = presentationData.moduleDetails.find(
+      (moduleRow) => moduleRow.moduleID === node.moduleID
+    )
+
+    if(!foundModule)
+      return;
+    if (!foundModule.hasLink)
+      return;
+
+    switch (settingsContext.openFileType) {
+      case OpenFileTypeEnum.XREF:
+        vscode.postMessage({
+          type: OpenFileTypeEnum.XREF,
+          columns: foundModule.moduleName,
```

Comment:
Why is module name here referred as 'columns'? To avoid confusion, I would name parameters for LISTING and XREF messages the same (name and lineNumber). Then you can also avoid this switch statement by just providing 'type' parameter from `settingsContext.openFileType`.